charts/nginx-ingress-services/values.yaml

```diff
@@ -40,6 +40,10 @@ certManager:
   #  (1) required to be set in values.yaml
   #  (2) used by Letsencrypt to send expiration notices and alike
   certmasterEmail:
+  # List of acme.cert-manager.io/v1.ACMEChallengeSolver to be used instead of a
+  # http01 nginx based solver. You can use secrets.certManager.customSolvers
+  # to store secrets required by the solvers.
```

Comment:
This is only one secret, and you're supposed to specify the contents below `.data` of it. We should probably clarify this in the comment here.

Reply:
Done